.github/workflows/test.yml

```diff
@@ -145,16 +145,11 @@ jobs:
             **/**.go
             go.mod
             go.sum
-      - uses: actions/setup-go@v4
+      - uses: actions/setup-go@v5
         with:
-<<<<<<< HEAD
-          go-version: 1.21.x
-      - name: Install GaiaV10
-=======
           go-version: 1.20.x
```

Comment:
probably the wrong version here. all the rest is using 1.21.x

Reply:
Pls notice how this test works and check the `Makefile` of v14.

V14 requires having go1.20 available, otherwise it would not build making the test fail.

I changed the Makefile constraint to be able to support `minimal_go_version` instead of exactly matching the required version.